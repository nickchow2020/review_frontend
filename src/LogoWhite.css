.appLogoHome{
  width: 13%;
  margin-left: auto;
  margin-right: auto;
  margin-top: 2rem;
}