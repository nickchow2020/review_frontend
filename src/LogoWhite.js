import React from "react"
import {ReactComponent as Logo} from "./thelogowhite.svg"
import "./LogoWhite.css"

const MyLogo = ()=>{
    return(
        <>
            <Logo className="appLogoHome"/>
        </>
    )
}

export default MyLogo