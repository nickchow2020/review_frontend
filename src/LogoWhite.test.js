import React from 'react';
import { render } from '@testing-library/react';
import LogoWhite from "./LogoWhite";

it ("should render", function(){
  render(<LogoWhite />)
})


it ("should match snapshot", function(){
  const {asFragment} = render(<LogoWhite />)
  expect(asFragment()).toMatchSnapshot()
})