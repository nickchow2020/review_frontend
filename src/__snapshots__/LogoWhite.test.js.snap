// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<DocumentFragment>
  <svg
    class="appLogoHome"
  >
    thelogowhite.svg
  </svg>
</DocumentFragment>
`;
